addons/web_editor/views/snippets.xml

```diff
@@ -384,6 +384,16 @@
         </we-row>
     </div>
 
+    <div data-js="LinkTool" data-selector="img, .media_iframe_video, span.fa, i.fa">
+        <we-input data-set-url="" name="url" title="Redirect to a URL when user clicks on the image"
+            string="Redirect to" class="o_we_large" placeholder="www.example.com"/>
+        <we-row>
+            <!-- The following checkbox should only appear when url field have something inside -->
+            <we-title class="o_long_title">⌙ Open in new window</we-title>
+            <we-checkbox data-name="link_on_image_opt" data-no-preview="true" data-set-new-window=""/>
+        </we-row>
```

Comment:
Don't care about "o_long_title" stuff for now, do it like the normal "open in new window field", we'll "fix" the crop issue in master. Here don't make a row and title etc, just a checkbox is needed.